### PR TITLE
Fix failing outerloop tests in System.Runtime.Extensions

### PR DESCRIPTION
**Pull request by maryamariyan**
[WIP] Need to run the CI to see if I missed any more failing tests

### CODE REVIEW

**Comment by maryamariyan** [comment]
@dotnet-bot test Outerloop

**Comment by maryamariyan** [comment]
@dotnet-bot test Outerloop Alpine.3.6 x64 Debug Build
@dotnet-bot test Outerloop CentOS.6 x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build
@dotnet-bot test Outerloop OSX x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop NETFX x64 Debug Build

**Comment by stephentoub** [comment]
There were tests failing in System.Diagnostics.Process as well.

Before eventually merging this, please make sure to run the outerloop tests a bunch of times, as failures here won't be deterministic.

**Comment by maryamariyan** [comment]
@dotnet-bot test Outerloop Alpine.3.6 x64 Debug Build
@dotnet-bot test Outerloop CentOS.6 x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build
@dotnet-bot test Outerloop OSX x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop NETFX x64 Debug Build

File: src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs [hunk 1]
```diff
@@ -251,12 +251,12 @@ private void Dispose(bool disposing)
             }
 
             // https://github.com/dotnet/corefx/issues/27366
```
Comment (ahsonkhan) [inline src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs]: Remove the issue in the comments once it has been resolved.

**Comment by maryamariyan** [comment]
@dotnet-bot test Outerloop Alpine.3.6 x64 Debug Build
@dotnet-bot test Outerloop CentOS.6 x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build
@dotnet-bot test Outerloop OSX x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop NETFX x64 Debug Build

**Comment by danmosemsft** [comment]
Results look good so far.

@dotnet-bot test Outerloop Alpine.3.6 x64 Debug Build
@dotnet-bot test Outerloop CentOS.6 x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build
@dotnet-bot test Outerloop OSX x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop NETFX x64 Debug Build

**Comment by danmosemsft** [comment]
Unrelated failures. One last time.

@dotnet-bot test Outerloop Alpine.3.6 x64 Debug Build
@dotnet-bot test Outerloop CentOS.6 x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build
@dotnet-bot test Outerloop OSX x64 Debug Build
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop NETFX x64 Debug Build

**Pull request merged**